packages/starlight/integrations/asides.ts

```diff
@@ -202,7 +203,7 @@ function remarkAsides(options: AsidesOptions): Plugin<[], Root> {
 						),
 						...titleNode,
 					]),
-					h('section', { class: 'starlight-aside__content' }, node.children),
+					h('div', { class: 'starlight-aside__content' }, node.children),
```

Comment:
I also removed the inner region landmark too as it doesn't bring any value imo. I think this [landmark definition](https://developer.mozilla.org/en-US/blog/aria-accessibility-html-landmark-roles/#the_region_landmark) sums it up well:

> Landmarks are meant to identify key areas of a page a user would most likely be interested in and would want to navigate to.